tools/go.mod

```diff
@@ -1,10 +1,15 @@
 module github.com/wata727/tflint/tools
 
-go 1.13
+go 1.12
```

Comment:
Go 1.13 adds version validations for `+incompatible` annotation. As a result, the terraform-provider-aws module version cannot be solved, so it is necessary to run it under Go 1.12 only under tools. See https://golang.org/doc/go1.13#modules

To solve this issue, terraform-provider-aws needs to be able to build with Go 1.13.
See also https://github.com/terraform-providers/terraform-provider-aws/issues/9992